src/components/SideBar/SidebarFooter.tsx

```diff
@@ -1,88 +1,75 @@
-import { Box, Typography } from "@mui/material";
-import React from "react";
-import { Link } from "react-router-dom";
+import { Box, Link } from "@mui/material";
+import { Link as Routerlink } from "react-router-dom";
 
 const styles = {
-  footerItem: {
-    textDecoration: "none",
-    paddingRight: 10,
-  },
-  footerItemText: {
-    "&:hover": {
-      textDecoration: "underline",
-    },
+  footerText: {
     color: "gray.main",
     fontSize: 12,
+    paddingRight: 1.5,
   },
   footerNav: {
     display: "flex",
     flexWrap: "wrap",
-    paddingX: 2,
+    justifyContent: "center",
   },
 };
 
 const SidebarFooter = () => {
   return (
-    <Box component={"nav"} sx={styles.footerNav}>
+    <Box component="nav" sx={styles.footerNav}>
       <Link
```

Comment:
I rather have links in the footer open a new tab rather than open in the new tab because of the UX in our app. 

We should not force users to lose their place on our website, because they could be scrolling far and navigating back might make it more difficult to find that place.

Reply:
Agreed